lib/libutils/ext/arch/riscv/mcount_rv.S

```diff
@@ -0,0 +1,127 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2023 Andes Technology Corporation
+ * Copyright (c) 2016, Linaro Limited
+ */
+
+#include <asm.S>
+
+#if defined(CFG_TA_GPROF_SUPPORT) || defined(CFG_FTRACE_SUPPORT)
```

Comment:
Does this patch-set enables support for gprof too on RISC-V? If not we should include corresponding `CFG_` checks.

Reply:
@b49020 Gprof support is future work.
I will remove related configurations first and add them back once RISC-V supports it.
Thanks for recommendation!

Reply:
Hi @b49020 

Commit https://github.com/OP-TEE/optee_os/pull/6129/commits/32552034144eb0d79cbd0142168cccb2a46723b6 removes `CFG_TA_GPROF_SUPPORT`.
Please check it.